components/service-operator/apis/storage/v1beta1/s3_types_test.go

```diff
@@ -0,0 +1,157 @@
+package v1beta1_test
+
+import (
+	"encoding/base64"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/alphagov/gsp/components/service-operator/apis/storage/v1beta1"
+	"github.com/alphagov/gsp/components/service-operator/internal/aws/cloudformation"
+	"github.com/alphagov/gsp/components/service-operator/internal/env"
+)
+
+var _ = Describe("S3Bucket", func() {
+
+	var o v1beta1.S3Bucket
+	var tags []cloudformation.Tag
+
+	BeforeEach(func() {
+		os.Setenv("CLUSTER_NAME", "xxx") // required for env package
+		o = v1beta1.S3Bucket{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "example",
```

Comment:
Can we replace the hardcoded value "example" with a constant variable `s3_object_name`?